charts/node/templates/statefulset.yaml

```
@@ -655,11 +688,20 @@ spec:
             - mountPath: /relaychain-keystore
               name: relaychain-keystore
             {{- end }}
+          {{- range $keys := .Values.node.extraConfigmapMounts }}
```

The $keys variable is unused here, you can replace by;

```suggestion
          {{- range .Values.node.extraConfigmapMounts }}
```